core/tests/protractor_desktop/ExplorationTranslationTab.js

```diff
@@ -131,11 +131,71 @@ describe('Exploration translation', function() {
     explorationEditorPage.saveChanges();
 
     explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
     explorationEditorTranslationTab.expectNumericalStatusToMatch(
       '(0/8)');
+
+    // click upload button
+    explorationEditorTranslationTab.openUploadAudioModal();
+    var audioToUpload = '../data/cafe.mp3';
+    var audioAbsolutePath = path.resolve(__dirname, audioToUpload);
+    var audioElem = element(by.css('[ng-class="inputFieldClassName"]'));
```

Comment:
Move this to utils file and create a new class instead.

Reply:
Maye, we can have uploadAudio method in ExplorationEditorTranslationTab.js (I see repeated code below)

Reply:
done